CHANGELOG.md

```diff
@@ -5,7 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [vNext]
+## [2.36.0]
+### Added
+-  Testing utilities 
```

Comment:
Could we simplify it in 1 line? For instance, `Benchmarking and autogenerated methods weights for `ddc-staking` pallet`?

Reply:
updated